test/image/mocks/bar_attrs_relative.json

```diff
@@ -2,27 +2,37 @@
     "data":[
         {
             "width":[1,0.8,0.6,0.4],
+            "text":[1,2,3333333333,4],
+            "textposition":"outside",
             "y":[1,2,3,4],
             "x":[1,2,3,4],
             "type":"bar"
         }, {
             "width":[0.4,0.6,0.8,1],
+            "text":["Three",2,"outside text",0],
```

Comment:
Two questions here:
1. why is "outside text" outside? Wouldn't it be just as big on the inside?
2. Where did "0" go? Even though the bar has no size, because it's the last bar we *could* show it where the last bar would have been (and likewise, it's confusing that the "4" from trace 0 shows up on the outside, it feels like only trace 1 should be able to have a label there.